packages/core/src/modules/connections/DidExchangeProtocol.ts

```diff
@@ -222,9 +222,9 @@ export class DidExchangeProtocol {
     if (routing) {
       services = this.routingToServices(routing)
     } else if (outOfBandRecord) {
-      const inlineServices = outOfBandRecord.outOfBandInvitation.services.filter(
-        (service) => typeof service !== 'string'
-      ) as OutOfBandDidCommService[]
+      const inlineServices = outOfBandRecord.outOfBandInvitation
+        .getServices()
+        .filter((service) => typeof service !== 'string') as OutOfBandDidCommService[]
```

Comment:
maybe we can also add a `getDidServices` and `getInlineServices` (or `getDidCommServices`). This way we don't have to do the string filter here, and we already get the correct type back (string[])